core/src/luigi-config/navigation/static-navigation-model.js

```diff
@@ -2080,6 +2080,34 @@ export function getStaticRootNodes(
         },
       ],
     },
+    {
+      category: i18next.t('configuration.title'),
```

Comment:
this view should be added only if extensibility feature is enabled

Reply:
indeed it should